utils/apiLog.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * /utils/apiLog.js
```

Comment:
Then maybe create a folder activityLogs and put all the logging files together there? Just trying to prevent having too many root level files (similar kind) in utils folder.